worlds/tunic/__init__.py

```diff
@@ -273,6 +291,13 @@ def create_items(self) -> None:
             items_to_create["Fool Trap"] += items_to_create[money_fool]
             items_to_create[money_fool] = 0
 
+        # creating these after the fool traps are made mostly so we don't have to mess with it
+        if self.options.breakable_shuffle:
+            for _, loc_data in breakable_location_table.items():
```

Comment:
Should be able to just loop over the values instead
```suggestion
            for loc_data in breakable_location_table.values():
```